### DISCUSSION TITLE
Can't pair Schneider Wiser powertag R9M60 (Resi9 F63A 1P+N)

### DISCUSSION THREAD

**Discussion by eekamouze** [Koenkk/zigbee2mqtt#17654] [Devices]
Hi,
I've one of these Schneider Electric powertag and was curious about connecting it to z2m. I was not expecting it to work out of the box as it is not in the list of supported devices (similar one but 3 phase is supported) but at least I thought I would be able to pair it. The only log I see (debug) when the device is in paraing mode:
`Received Zigbee message from 'Coordinator', type 'commandNotification', cluster 'greenPower', data '{"commandID":227,"frameCounter":136,"srcID":0}' from endpoint 242 with groupID null, ignoring since it is from coordinator`

Any idea?
Thanks

**Comment by senna1992** [comment]
It's the chip. 
Get yourself a "sonoff - dongle typ p". it will pair (zgp unit doesnt pair it just appears in your device list)

Currently facing some issues because of the a9mem1570 converter. (and inverted pwoer values due to upstream installation)


https://www.reddit.com/r/homeassistant/comments/17t1ip7/progress_powertag_r920m_working_under/

**Reply by killernils** [reply]
Do you know if this is still the case? Looks like the guy in the reddit post didnt get any power values?
He mention something about a second Wiser device acting as translators?
Does that mean that I cant only use the Sonoff-dongle and the ([supported](https://www.zigbee2mqtt.io/devices/A9MEM1570.html)) A9MEM1570 powertag directly?
